packages/nimble-components/src/button/tests/button-matrix.stories.ts

```diff
@@ -81,3 +81,11 @@ export const hiddenButton: StoryFn = createStory(
 export const textCustomized: StoryFn = createMatrixThemeStory(
     textCustomizationWrapper(html`<${buttonTag}>Button</${buttonTag}>`)
 );
+
+export const inlineAlignment: StoryFn = createStory(
+    html`Text
+        <${buttonTag}>Button</${buttonTag}>
+        Text
+        <${buttonTag}><${iconKeyTag} slot="start"></${iconKeyTag}>Button</${buttonTag}>
+        <img src="https://github.com/ni/nimble/raw/main/docs/nimble-logo-icon.svg" width="50px" style="border: 1px black solid">`
```

Comment:
Don't couple to other arbitrary resources not owned by the test for no reason. If we move or rename that logo that would change the behavior of the tests.

Can you just do something like:

```suggestion
        <div style="display:inline-block; width:50px; height:50px; border: 1px black solid;"></div>`
```